plugin/forward/README.md

```diff
@@ -91,6 +91,7 @@ forward FROM TO... {
     The flag is default `true`.
   * `domain FQDN` - set the domain name used for health checks to **FQDN**.
     If not configured, the domain name used for health checks is `.`.
+  * `on_fail ONFAIL` - Allowed values for ONFAIL are **servfail** or **spray**. If healthcheck is broken for an upstream the setting servfail would return servfail response to clients and would not contact the upstreams whereas spray would keep health checking the upstreams even if upstreams are failing. It defaults to **spray**.
```

Comment:
* the description of how spray works is incorrect.  
* health checking always occurs when an upstream is unhealthy, regardless of this setting. 
* this new `on_fail` setting only comes into effect when _all_ upstreams are down, not just any one. 
 
Please use the following description instead:

```suggestion
  * `on_fail ONFAIL` - controls how requests are handled when _all_ upstream servers are unhealthy and unresponsive to health checks. Allowed values for **ONFAIL** are `servfail` and `spray`.  `servfail` will immediately return SERVFAIL responses for all requests. `spray` will instead send requests to a random upstream.  The default behavior is `spray`.
```

Reply:
reworded the description as per your suggestion